cmd/test/main.go

```diff
@@ -244,6 +244,10 @@ func GenerateTestCommandData(testCommandFlags *TestCommandFlags, localConfigCont
 		k8sVersion = evaluationPrerunDataResp.DefaultK8sVersion
 	}
 
+	if k8sVersion == "" {
```

Comment:
Maybe we can refactor it to be in the first `if k8sVersion == "" `?

Reply:
it doesn't make sense, the priority goes env var > server response > default